pom.xml

```diff
@@ -26,7 +25,7 @@
 
     <groupId>com.connexta.search</groupId>
     <artifactId>index-api</artifactId>
-    <version>0.2.1-SNAPSHOT</version>
+    <version>0.4.0-SNAPSHOT</version>
```

Comment:
:question: Is we release 0.3.0, should this be 0.3.1-SNAPSHOT?  Same comment for other pom versions below.

Reply:
0.3.0 should be released before this PR is merged. That's why this PR is marked as a draft.